docs/development/nodejs/how-to-install-nodejs-and-nginx-on-ubuntu-18-04/index.md

```diff
@@ -0,0 +1,257 @@
+og_description: 'In this guide you will install and configure NGINX to serve static site content on an Ubuntu 18.04 Linode. You will also create a Node.js server and use NGINX as a reverse proxy to your Node.js server. To test your configurations, you will create an index.html file as your static content and a test JavaScript file to be served by your Node.js server.'
+keywords: ["linode guide", "hosting a website", "website", "linode setup", " install node.js", " install nginx", "ubuntu", " front-end requests", " back-end requests"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-04-14
+modified_by:
+    name: Linode
+published: 2020-04-14
+title: 'How to Install Node.js and NGINX on Ubuntu 18.04'
+h1_title: 'Installing Node.js and NGINX on Ubuntu 18.04'
+external_resources:
+ - '[Node.js](http://nodejs.org)'
+ - '[NGINX](http://nginx.com/)'
+ - '[NodeSchool](http://nodeschool.io/)'
+ - '[Node Version Manager](https://github.com/creationix/nvm)'
+ - '[npm](https://www.npmjs.com/)'
+audiences: ["intermediate"]
+concentrations: ["Web Applications"]
+languages: ["javascript"]
+---
+
+Node.js is an open-source JavaScript runtime environment that can serve dynamic and responsive content and is often used to create and serve web applications. When serving Node.js applications, NGINX is commonly used to create a reverse proxy that points at a running Node.js server. In this guide, you will install and configure NGINX to handle requests to static files, like `index.html` and also, create a reverse proxy to a Node.js server. You will then create a test JavaScript file in order to test your running Node.js server.
+
+## Before You Begin
+
+1.  If you want to use a custom domain name for your site, purchase a domain name from a trusted registrar and use Linode's [DNS Manager](/docs/platform/manager/dns-manager/) to [add the domain](/docs/platform/manager/dns-manager/#add-a-domain) and [create a domain record](/docs/platform/manager/dns-manager/#add-dns-records) for it.
+
+1.  Set up your Linode using the [Getting Started](/docs/getting-started/) and [Securing your Server](/docs/security/securing-your-server/) guides.
+
+    {{< note >}}
+Don't forget to update your Linode's `/etc/hosts` file with its public IP address and your site's fully qualified domain name, as explained in the [Update Your System's hosts File](/docs/getting-started/#update-your-system-s-hosts-file) section of the [Getting Started](/docs/getting-started/) guide.
+    {{</ note >}}
+
+
+    {{< content "limited-user-note-shortguide" >}}
+
+## Install and Configure NGINX
+
+1.  Install NGINX and the screen utility. You will use screen in the [Create Your the Node.js Web Server File](#create-your-the-node-js-web-server-file).
```

Comment:
Create the, no your. Will also need to update link

Reply:
updated!